bin/mana_launch

```diff
@@ -112,7 +112,7 @@ fi
 # A temporary workaround is to swap the memory addresses of NIMROD and lh_proxy,
 # which requires a specially built version of lh_proxy.
 app_name="`echo $options | head -n1 | xargs basename`"
-if [ "$app_name" == "nimrod" ]; then
+if [[ $app_name == *"nimrod"* ]]; then
```

Comment:
@JainTwinkle,
    Thanks for catching this bug in MANA-161.  Great catch.  I have a suggested alternate fix.

I think you are using wild cards (`*`) to look for a file name with the substring 'nimrod'.  That depends on looking in the local directory, and might depend on whether the user's PATH includes `.`.

What about something like:
```
if echo $app_name | grep nimrod; then
```
I think that works, and it is less dangerous.